src/gui/properties/propertieswidget.cpp

```diff
@@ -313,7 +313,7 @@ void PropertiesWidget::loadTorrentInfos(BitTorrent::Torrent *const torrent)
     // Save path
     updateSavePath(m_torrent);
     // Hash
-    m_ui->labelHashVal->setText(m_torrent->hash().toString());
+    m_ui->labelHashVal->setText(m_torrent->id().toString());
```

Comment:
We should show both SHA1 and SHA256 in the GUI, in a later PR of course.
Maybe you could add a `TODO` here?

Reply:
>We should show both SHA1 and SHA256 in the GUI

Do you think that this will bring some benefit to the user? Okay, let's discuss this later.

Reply:
My 2c: we should display all hashes that a torrent contains - including the truncated SHA256, if applicable (perhaps under a label of "V1-compatible hash: `deadbeef`").

Reply:
>TorrentID isn't useful for the users as it is just an internal id in qbt, libtorrent.

Trackers don't use anything but it.

Reply:
>What about pure v2 torrents?

`m_torrent->hash().sha1().toString()` will return empty and there will be another `m_torrent->hash().sha2-256().toString()` to access it. However, you don't really handle bittorrent v2 cases in this PR anyway, so can provide only `sha1()` part.

>Trackers don't use anything but it.

Users don't care about the protocol level but they are about the info stored inside the .torrent file (or magnet metadata).

Reply:
Following https://github.com/qbittorrent/qBittorrent/pull/14503#discussion_r591689474 please add a TODO here too.

Reply:
>Following https://github.com/qbittorrent/qBittorrent/pull/14503#discussion_r591689474 please add a TODO here too.

Done.